### PR TITLE
Manitoba Pension income amount

### LINKED ISSUE #302: Manitoba Pension income amount
[Reference](https://www.canada.ca/en/revenue-agency/services/forms-publications/tax-packages-years/general-income-tax-benefit-package/manitoba/5007-pc/information-residents-manitoba.html#P70_5092)

### PR DESCRIPTION
**Pull request by PavelMakarchuk**
Fixes #302

### CODE REVIEW

File: policyengine_canada/variables/household/income/individual/pension_income.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from policyengine_canada.model_api import *
+
+
+class pension_income(Variable):
+    value_type = float
+    entity = Person
+    label = "Pension Income"
+    unit = CAD
+    documentation = "Income from the canada pension plan"
```
Comment (MaxGhenis) [inline policyengine_canada/variables/household/income/individual/pension_income.py:9]: What about private pension income?
```suggestion
    documentation = "Income from the Canada pension plan"
```

Comment (PavelMakarchuk) [reply policyengine_canada/variables/household/income/individual/pension_income.py:9]: Should we use the pension and savings plan income variable instead?

Comment (MaxGhenis) [reply policyengine_canada/variables/household/income/individual/pension_income.py:9]: If that's what the law refers to ;)

Comment (PavelMakarchuk) [reply policyengine_canada/variables/household/income/individual/pension_income.py]: Found it! Absolutely right

File: policyengine_canada/parameters/gov/provinces/mb/tax/income/credits/pension_amount/max_amount.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+description: Monitabo provides this maximum amount under the Pension amount credit.
```
Comment (MaxGhenis) [inline policyengine_canada/parameters/gov/provinces/mb/tax/income/credits/pension_amount/max_amount.yaml:1]:
```suggestion
description: Manitoba provides this maximum amount under the Pension amount credit. 
```

**Pull request merged**